cppForSwig/cryptopp/Makefile

```diff
@@ -68,11 +68,12 @@ else
 ifeq ($(GAS219_OR_LATER),0)
 CXXFLAGS += -DCRYPTOPP_DISABLE_AESNI
 else
-ifeq ($(XCOMPILE),0)
-CXXFLAGS += -mpclmul -mssse3 -maes -msse4
-else
+ifeq ($(CXX),arm-linux-gnueabihf-g++)
```

Comment:
Yikes, this is really brittle. An autoconf test could check for working -m flags.